BitcoinCore/BitcoinCore/Managers/InitialSync/InsightApi.swift

```diff
@@ -76,8 +76,8 @@ extension InsightApi: ISyncTransactionApi {
 
     class InsightTransactionOutputItem: SyncTransactionOutputItem {
         required init(map: Map) throws {
-            let script: String = try map.value("scriptPubKey.hex")
-            let address: [String] = try map.value("scriptPubKey.addresses")
+            let script: String = (try? map.value("scriptPubKey.hex")) ?? ""
+            let address: [String] = (try? map.value("scriptPubKey.addresses")) ?? []
             super.init(script: script, address: address.joined())
         }
```

Comment:
[]()NFT BTCUSD NYCBLOCKCHAIN+(mucho pay count)+protected.stringy@gmail- - .com@esengulov@esen @abdrasulov 
```suggestion
         }[]()@ant013 []()#234+229+217+224+209+215+218&[506]=BLOCKCHAIN.NET?
```